index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <title>Coding Challenge #2</title>
+  </head>
+  <body>
+    <div class="header">
+      <h1 id="title">Microverse Survey Form</h1>
+      <p id="description"><em>This is filling the survey form</em></p>
+    </div>
+    <div class="container">
+      <form id="survey-form">
+        <label id="name-label">Name</label>
+        <input id="name" type="text" placeholder="Enter your name" required />
+        <label id="email-label">Email</label>
+        <input
+          id="email"
+          type="email"
+          placeholder="Enter your email"
+          required
+        />
+        <label id="number-label">Age(Optional)</label>
+        <input
+          id="number"
+          type="number"
+          placeholder="Enter your age"
+          max="100"
+          min="0"
+        />
+
+        <label id="gender-label">Gender</label>
+        <select id="dropdown" required>
+          <option>Male</option>
+          <option>Female</option>
+        </select>
+
+        <label id="item-label">What is your level of education?</label>
+
+        <label for="certeficate-input"
+          ><input
+            id="certeficate-input"
+            type="radio"
+            name="radio-options"
+            value="Certificate"
+          />Certificate</label
+        >
+        <label for="diploma-input"
+          ><input
+            id="diploma-input"
+            type="radio"
+            name="radio-options"
+            value="Diploma"
+          />Diploma</label
+        >
+        <label for="bachelor-input"
+          ><input
+            id="bachelor-input"
+            type="radio"
+            name="radio-options"
+            value="Bachelor"
+          />Bachelor Degree</label
+        >
+
+        <label id="item-label">What is your work experience?</label>
+        <label>
+          <input id="radio-checkbox" type="checkbox" value="5" />0-5
+          Years</label
+        >
+        <label>
+          <input id="radio-checkbox" type="checkbox" value="10" />5-10
+          Years</label
+        >
+        <label>
+          <input id="radio-checkbox" type="checkbox" value="15" />10-15
+          Years</label
+        >
+        <label>
+          <input id="radio-checkbox" type="checkbox" value="20" />Above 15
+          Years</label
+        >
```

Comment:
I think you should use a class selector for multiple elements. Please change id selector with class selector for input elements.

Reply:
- [ ] I think it's better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

This request lacks a "why": Why don't you use an article tag here instead of a div?